### PR TITLE
Update VocPrez prez:links structure for concept schemes and collections

### PR DESCRIPTION
**Pull request by edmondchuc**
Closes https://github.com/RDFLib/prez-ui/issues/31

### CODE REVIEW

File: prez/models/vocprez_item.py [hunk 1]
```diff
@@ -46,11 +46,11 @@ def populate(cls, values):
         elif url_parts[2] == "collection":  # collections
             values["curie_id"] = values.get("collection_curie")
             values["general_class"] = SKOS.Collection
-            values["link_constructor"] = f"/v/object?uri="
+            values["link_constructor"] = f"/v/vocab/{collection_curie}"
```
[the remark this answers is not in the document]
Comment (edmondchuc) [reply prez/models/vocprez_item.py:49]: I asked @jamiefeiss the same question in the Prez channel and he said this is the previous Prez's behaviour. My suggestion is to go with the original behaviour and we can discuss a behaviour change in a separate issue and implement in a separare PR. What do you think?

[the remark this answers is not in the document]
Comment (recalcitrantsupplant) [reply prez/models/vocprez_item.py:49]: see https://github.com/RDFLib/prez/discussions/94

**Comment by edmondchuc** [comment]
@recalcitrantsupplant @jamiefeiss, please review changes updating prez link for concepts within collections to `/v/collection/collection_id/concept_id`.

This should work for all concepts within collections and the logic is easier if we don't involve the legacy concept schemes behaviour. What determines `collection_id` here is dependent on which collection the user navigated to before reaching the concept.

The Prez UI code has been updated in https://github.com/RDFLib/prez-ui/pull/39 to support this change.

**Comment by recalcitrantsupplant** [comment]
Ran locally against IDN data ok (bar the issue in my PR)
Could you update the discussion https://github.com/RDFLib/prez/discussions/94 with details

**Review by recalcitrantsupplant — APPROVED** [review]

**Review by jamiefeiss — APPROVED** [review]
Works fine, tested with https://github.com/RDFLib/prez-ui/pull/39

**Comment by edmondchuc** [comment]
Thanks for the review!

**Pull request merged**